scripts/extraOptions

```diff
@@ -11,14 +11,14 @@ set -o pipefail
 EXTRA_OPTS=("")
 
 # Check if it is a scheduled build
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
-    ## Search for snapshots is harcdoded to scala 2.13
-    ## Queries differ because the artifact names changed during the 2.6 milestones phase.
-    AKKA_25_SNAPSHOT=$(curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -oEi '2\.5-[0-9]{8}-[0-9]{6}' | sort | tail -n 1)
-    AKKA_26_SNAPSHOT=$(curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -oEi '2\.6\.[0-9]+\+[0-9]+-[0-9a-f]{8}' | sort -V | tail -n 1)
+#if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
```

Comment:
I'll revert this change, i just wanted to see the nightlies work without waiting for the cron